SPECS/pam_wrapper/pam_wrapper.spec

```diff
@@ -0,0 +1,208 @@
+Summary:        A tool to test PAM applications and PAM modules
+Requires:       pam_wrapper = %{version}-%{release}
+
+%description -n libpamtest
+If you plan to test a PAM module you can use this library, which simplifies
+testing of modules.
+
+%package -n libpamtest-devel
+Summary:        A tool to test PAM applications and PAM modules
+Requires:       libpamtest = %{version}-%{release}
+Requires:       pam_wrapper = %{version}-%{release}
+Recommends:     cmake
+Recommends:     pkg-config
+
+%description -n libpamtest-devel
+If you plan to develop tests for a PAM module you can use this library,
+which simplifies testing of modules. This sub package includes the header
+files for libpamtest.
+
+%package -n libpamtest-doc
+Summary:        The libpamtest API documentation
+
+%description -n libpamtest-doc
+Documentation for libpamtest development.
+
+%package -n python3-libpamtest
+Summary:        A python wrapper for libpamtest
+Requires:       libpamtest = %{version}-%{release}
+Requires:       pam_wrapper = %{version}-%{release}
+
+%description -n python3-libpamtest
+If you plan to develop python tests for a PAM module you can use this
+library, which simplifies testing of modules. This subpackage includes
+the header files for libpamtest
+
+%prep
+%autosetup -S git
+
+
+%build
+if test ! -e "obj"; then
+  mkdir obj
+fi
+pushd obj
+%cmake \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  -DUNIT_TESTING=ON \
+  %{_builddir}/%{name}-%{version}
+
+make %{?_smp_mflags} VERBOSE=1
+make doc VERBOSE=1
+popd
+
+
+%install
+pushd obj
+make DESTDIR=%{buildroot} install
+popd
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+%post -p /sbin/ldconfig -n libpamtest
+%postun -p /sbin/ldconfig -n libpamtest
+
+%check
+pushd obj
+ctest --output-on-failure
+popd
+
+%files
+%{_libdir}/libpam_wrapper.so*
+%{_libdir}/pkgconfig/pam_wrapper.pc
+%dir %{_libdir}/cmake/pam_wrapper
+%{_libdir}/cmake/pam_wrapper/pam_wrapper-config-version.cmake
+%{_libdir}/cmake/pam_wrapper/pam_wrapper-config.cmake
+%{_libdir}/pam_wrapper/pam_chatty.so
+%{_libdir}/pam_wrapper/pam_matrix.so
+%{_libdir}/pam_wrapper/pam_get_items.so
+%{_libdir}/pam_wrapper/pam_set_items.so
+%{_mandir}/man1/pam_wrapper.1*
+%{_mandir}/man8/pam_chatty.8*
+%{_mandir}/man8/pam_matrix.8*
+%{_mandir}/man8/pam_get_items.8*
+%{_mandir}/man8/pam_set_items.8*
+
+%files -n libpamtest
+%{_libdir}/libpamtest.so.*
+
+%files -n libpamtest-devel
+%{_libdir}/libpamtest.so
+%{_libdir}/pkgconfig/libpamtest.pc
+%dir %{_libdir}/cmake/pamtest
+%{_libdir}/cmake/pamtest/pamtest-config-relwithdebinfo.cmake
+%{_libdir}/cmake/pamtest/pamtest-config-version.cmake
+%{_libdir}/cmake/pamtest/pamtest-config.cmake
+%{_includedir}/libpamtest.h
+
+%files -n libpamtest-doc
+%doc obj/doc/html
+
+%files -n python3-libpamtest
+%{_lib}/python3.7/site-packages/pypamtest.so
+
+%changelog
+* Mon Jul 26 2021 Shane Guan <shaneguan@microsoft.com> - 1.1.3-4
+- Replace ldconfig_scriptlets with the expanded form
+- Replace gpgv2 with gpg
```

Comment:
And similarly, last comment about us not using `gpg` anymore.;) Please check, where else we might still have this leftover.